foundation-extensions/mobile-core/lifecycle/lifecycle-api-reference.md

```diff
@@ -333,13 +327,7 @@ ACPCore.lifecyclePause()
 {% tab title="React Native" %}
 #### JavaScript
 
-> Note: Implementing Lifecycle via JavaScript may lead to inaccurate Lifecycle metrics, therefore we recommend implementing Lifecycle in native Android and iOS code. However, these APIs are still provided in JavaScript to support flexible Lifecycle implementations.
-
-**Syntax**
-
-```jsx
-lifecyclePause();
-```
+> Note: Implementing Lifecycle via JavaScript may lead to inaccurate Lifecycle metrics, therefore we recommend implementing Lifecycle in native Android and iOS code. 
```

Comment:
Should we change to 
When using React Native, registering Lifecycle with Mobile Core should be done in native code which is shown under the Android and iOS(ACP 2.x) tabs.